composer.json

```diff
@@ -25,7 +25,7 @@
         "doctrine/cache": "^1.0",
         "doctrine/collections": "^1.0",
         "doctrine/event-manager": "^1.0",
-        "doctrine/reflection": "^1.0"
+        "doctrine/reflection": "^1.1"
```

Comment:
Noting that this is required for a fix. Since this transitive dependency fixes the issue for ORM, we've decided to bump this in a patch release.